packages/react-data-grid/src/common/editors/EditorPortal.tsx

```diff
@@ -1,43 +1,24 @@
-import React from 'react';
+import React, { useState, useLayoutEffect } from 'react';
 import ReactDOM from 'react-dom';
 
 interface Props {
   children: React.ReactNode;
-  target: Node;
+  target: Element;
 }
 
-interface State {
-  isMounted: boolean;
-}
-
-export default class EditorPortal extends React.Component<Props, State> {
+export default function EditorPortal({ target, children }: Props) {
   // Keep track of when the modal element is added to the DOM
-  readonly state: Readonly<State> = {
-    isMounted: false
-  };
-
-  el = document.createElement('div');
```

Comment:
I don't remember for what we reason we used an extra div here. Let me know if I should add it back.

Reply:
I believe that's how the instruction was documented when we first use the portal in a class component.

https://reactjs.org/docs/portals.html#event-bubbling-through-portals
see the modal example
![image](https://user-images.githubusercontent.com/25043398/57151958-ca1e8d80-6d97-11e9-87b0-821e1bd512c3.png)

Reply:
I do not think we need it